pkg/node/sendoperation/transaction/transaction.go

```diff
@@ -0,0 +1,61 @@
+package transaction
+
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/massalabs/thyra/pkg/node/base58"
+)
+
+type OperationDetails struct {
+	Amount           string `json:"amount"`
+	RecipientAddress []byte `json:"recipient_address"`
+}
+
+type Operation struct {
+	Transaction OperationDetails `json:"Transaction"`
+}
+
+type Transaction struct {
+	recepientAddress string
+	amount           uint64
+}
+
+func New(recepientAddress string, amount uint64) *Transaction {
+	return &Transaction{
+		recepientAddress: recepientAddress,
+		amount:           amount * 1e9,
```

Comment:
Why are you multiplying ?

Reply:
> Why are you multiplying ?

Multiplying because if you intent to send 1 MAS, the network will actually send 1e-9.